### PR TITLE
Quality of life update

### PR DESCRIPTION
**Pull request by XstreamSpeed**
**Please describe the changes this PR makes and why it should be merged:**
This PR makes many changes such as many new features, error checks and mostly grammatical errors. 

**Status and versioning classification:**
NodeJS v17.8.0, Discord.JS v13, branch v5

### CODE REVIEW

File: commands/slash/replay.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ const command = new SlashCommand()
       embeds: [
         new MessageEmbed()
           .setColor(client.config.embedColor)
-          .setDescription(`Replay [${song.title}](${song.uri})`),
+          .setTitle(`Now replaying - [${song.title}](${song.uri})`),
```
Comment (DarrenOfficial) [inline commands/slash/replay.js:48]: change to setDescription

**Review by DarrenOfficial — CHANGES_REQUESTED** [review]

**Review by DarrenOfficial — CHANGES_REQUESTED** [review]
Remove this line entirely, there's no use for this.

File: commands/slash/247.js [hunk 2]
```diff
@@ -41,6 +41,7 @@ const command = new SlashCommand()
       const embed = client.Embed(`✅ | **24/7 mode is now on.**`);
       return interaction.reply({ embeds: [embed] });
     }
+      if(err) console.log(err) && client.errorEmbed("An error has occurred, please vist console for more information!")
```
Comment (DarrenOfficial) [inline commands/slash/247.js:44]: *Remove this line entirely there is no use for this

**Review by DarrenOfficial — CHANGES_REQUESTED** [review]

File: commands/slash/replay.js (hunk 1 above)
Comment (XstreamSpeed) [reply commands/slash/replay.js:48]: Alr

File: commands/slash/247.js (hunk 2 above)
Comment (XstreamSpeed) [reply commands/slash/247.js:44]: Below it was a text saying it has no checks for errors so I added one

Comment (DarrenOfficial) [reply commands/slash/247.js:44]: it does has checks for errros?

Comment (DarrenOfficial) [reply commands/slash/247.js:44]: oh I see what you're talking about, that one has been solved.

Comment (DarrenOfficial) [reply commands/slash/247.js:44]: still remove that tho, you didn't define "err" therefore it's just a zombie code

Comment (XstreamSpeed) [reply commands/slash/247.js:44]: alr np

**Pull request closed without merging**